README.md

```diff
@@ -111,91 +84,60 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://react-capstone-d0aw.onrender.com)
```

Comment:
- [ ] For some reason, the live demo link you provided in your README is not loading. Kindly look into why this is not the case or maybe deploy the application to [netlify](https://www.netlify.com/) instead.

![image](https://user-images.githubusercontent.com/86318284/217731497-4889a321-9c16-41cc-8229-6de1350d9695.png)